dockerfiles/ci/buster/Dockerfile

```diff
@@ -81,22 +82,24 @@ RUN set -eux; \
     \
 # Allow nginx to be run as non-root for tests
     chown -R circleci:circleci /var/log/nginx/ /var/lib/nginx/; \
-# JSON parser
-    export JQ_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/jq-latest"; \
-    curl --silent --show-error --location --fail --retry 3 --output /usr/bin/jq $JQ_URL; \
-    chmod +x /usr/bin/jq; \
-    jq --version; \
-    \
+# Install CMake
+    cd /tmp && curl -OL https://github.com/Kitware/CMake/releases/download/v3.19.6/cmake-3.19.6-Linux-x86_64.tar.gz; \
+    (echo "139580473b84f5c6cf27b1d1ac84e9aa6968aa13e4b1900394c50075b366fb15 cmake-3.19.6-Linux-x86_64.tar.gz" | sha256sum -c -); \
+    mkdir -p /opt/cmake/3.19.6 && cd /opt/cmake/3.19.6; \
+    tar -xf /tmp/cmake-3.19.6-Linux-x86_64.tar.gz --strip 1; \
+# Install Catch2
```

Comment:
New catch2 version released 2 days ago as well.

Reply:
Nice _catch_. 😉 This is updated.